sys/shell/commands/sc_netif.c

```diff
@@ -103,7 +104,7 @@ static void _hl_usage(char *cmd_name)
 
 static void _flag_usage(char *cmd_name)
 {
-    printf("usage: %s <if_id> [-]{promisc|autoack|csma|autocca|preload|iphc|rtr_adv}\n", cmd_name);
+    printf("usage: %s <if_id> [-]{promisc|autoack|csma|autocca|cca_threshold|preload|iphc|rtr_adv}\n", cmd_name);
```

Comment:
Accidentally added this in my PR to yours. As the function name "_flag_usage" says, only flags should be listed. Since cca_threshold is not a flag, it should maybe not printed here.